src/ProjNet/CoordinateSystems/Projections/LambertAzimuthalEqualAreaProjection.cs

```diff
@@ -7,7 +7,7 @@ namespace ProjNet.CoordinateSystems.Projections
     /// <summary>
     /// 
     /// </summary>
-    public class LambertAzimuthalEqualAreaProjection : MapProjection
+    internal class LambertAzimuthalEqualAreaProjection : MapProjection
```

Comment:
@jarobbins, could you reverse this, as it imposes a breaking change. Other than that, you are right.

Reply:
LambertAzimuthalEqualAreaProjection is public again.